<launch>
  <node pkg="seatrac" type="seatrac_node" name="usbl">
        <param name="PortName" value="/dev/ttyS0" />
        <param name="master" value="true" />
        <param name="usbl" value="true" />
        <param name="auto_mode" value="true" />
        <param name="only_ack" value="true" />
        <param name="vehicle_ahrs" value="true" />
        <param name="timeout" value="5" />
        <param name="id" value="1" />
        <param name="track_ids" value="[2,3]" />
  </node>
</launch>
